src/BenchmarkDotNet/Engines/Engine.cs

```diff
@@ -138,6 +141,8 @@ public Measurement RunIteration(IterationData data)
             if(!isOverhead)
                 IterationSetupAction();
 
+            ClearCache();
```

Comment:
please add a comment explaining why it's before  `GcCollect` (I assume that because one of the strategies can allocate managed memory?)

Reply:
I wasn't aware of it. I wanted to do it after `IterationSetupAction()`. I'll add message.